src/create/index.js

```diff
@@ -80,17 +83,21 @@ function addService(opts) {
 			});
 		})
 		.then(() => {
-			const templatePath = path.join(__dirname, "service.template");
+			const templatePath = _typescript ? path.join(__dirname, "typescript.service.template"):path.join(__dirname, "service.template");
 			const template = fs.readFileSync(templatePath, "utf8");
 
-			return new Promise((resolve, reject) => {
-				render(template, values, function (err, res) {
+			return new Promise( (resolve, reject) => {
+				render(template, values, async function (err, res) {
 					if (err)
 						return reject(err);
 
-					const { newServicePath } = values;
+					const { serviceFolder , serviceName  } = values;
+					const newServicePath =  path.join(serviceFolder, `${serviceName}.service${_typescript ?".ts" :".js"}`);
+
+
+					console.log("newServicePath", newServicePath);
 					console.log(`Create new service file to '${newServicePath}'...`);
-					fs.writeFileSync(path.resolve(newServicePath), res, "utf8");
```

Comment:
Don't change it, because it requires Node 10, and in this case it causes breaking change. `writeFileSync` is enough here.

Reply:
Btw:

Error: Cannot find module 'fs/promises'
Require stack:
- D:\Work\moleculer\moleculer-cli\src\create\index.js
- D:\Work\moleculer\moleculer-cli\bin\moleculer.js

Reply:
okay, removed